site/en/guide/migrate.ipynb

```diff
@@ -2159,7 +2159,6 @@
     },
     "kernelspec": {
       "display_name": "Python 3",
-      "language": "python",
```

Comment:
Why delete this?

If colab is adding this, it will always be added.

Reply:
This is a result of c/322176921
Colab is not adding it. Other environments are and it's not necessary